### DISCUSSION TITLE
A ton of "not found" errors out of nowhere

### DISCUSSION THREAD

**Discussion by NikRetaNCAM** [HaxeFlixel/flixel#2593] [Q&A]
![image](https://user-images.githubusercontent.com/60806448/176989278-0215c114-97ed-4430-8ff7-8a0fcc2903bc.png)
i have no idea why this is happening. yesterday it all worked fine, i turned my pc off and today it's like this

**Comment by Geokureli** [comment]
can you show your folder structure and the `<source path="source">` tag in your project.xml? Here is an example of mine:
<img width="786" alt="Screen Shot 2022-07-02 at 12 10 11 PM" src="https://user-images.githubusercontent.com/2609513/177009913-0554d12e-5bc1-44e1-a81b-d2de4f752efb.png">

Also, what is your PlayState's package?
<img width="290" alt="Screen Shot 2022-07-02 at 12 13 55 PM" src="https://user-images.githubusercontent.com/2609513/177010047-107baa54-c5c1-4258-88b6-eb76d06d31d5.png">


based on the top error, it looks like you're source path is set the folder containing "source" rather than source itself. the compiler seems to be confusing PlayState in the root package the "source" folder , vs `source.PlayState` where source is a package

**Reply by NikRetaNCAM** [reply]
![image](https://user-images.githubusercontent.com/60806448/177026010-f2cd4877-eaef-4b12-9a74-8d381ccd002e.png)
there was no "source path" line, i wrote that just now

also, shall i add, that upon opening a project folder, an error message saying "lime completion setup failed. is the lime command available? (note: it is). Try running lime setup command or changing the lime.executable setting" pops up
![image](https://user-images.githubusercontent.com/60806448/177026061-ccced149-2c9c-486c-996e-3dcde8789d6e.png)

**Reply by Geokureli** [reply]
what happened when you added the source? do builds work now?

**Reply by NikRetaNCAM** [reply]
> what happened when you added the source? do builds work now?

no.

**Reply by NikRetaNCAM** [reply]
> > what happened when you added the source? do builds work now?
> 
> no.

gotta mention - all the stuff i described above does not apply to projects i created myself

**Reply by Geokureli** [reply]
what happens when you build via command line
